doc/data/messages/s/suppressed-message/bad.py

```diff
@@ -0,0 +1 @@
+test = ["a" "b"]  # pylint: disable=implicit-str-concat # [suppressed-message]
```

Comment:
I'd prefer to do this with a `pylintrc` that enables the message and then disable it here.

The enable in the running of the tests is just additional "headspace" that gets occupied when dealing with these tests.
The fact that we had so many issues with just disabling the cleaning of messages after each file shows that neither of us (or other maintainers) fully understand all the implications of the functional test runners.

Reply:
I tried that too (and also enable in file) but I think pylintrc is the baseline that is modified by the command line argument. Not sure if it's possible.

Reply:
I agree. I tried to use disable-next to enable the suppressed message and that did not work for me

Reply:
Yeah, not sure why, I think the right order for message control should be "configuration < commande line < file directive according to scope":
```bash
$ cat a.py
# pylint: enable=invalid-name
tEsT = "apple"  # pylint: disable=invalid-name
$ pylint a.py --disable=all --enable=invalid-name,suppressed-message,astroid-error,syntax-error
************* Module a
a.py:2:0: I0020: Suppressed 'invalid-name' (from line 2) (suppressed-message)
--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
$ pylint a.py --disable=all --enable=suppressed-message,astroid-error,syntax-error 
--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```
Clearly fixing the message control is a task that is not going to be tackled like a young ballerina by Sébastien Chabal.